avalon/maya/commands.py

```diff
@@ -33,17 +34,29 @@ def reset_frame_range():
         cmds.warning("No edit information found for %s" % shot["name"])
         return
 
-    fps = {
-        "12": "12fps",
-        "15": "game",
-        "16": "16fps",
-        "24": "film",
-        "25": "pal",
-        "30": "ntsc",
-        "48": "show",
-        "50": "palf",
-        "60": "ntscf"
-    }.get(api.Session.get("AVALON_FPS"), "pal")  # Default to "pal"
+    # this handles fps set as decimal number. For example
+    # fps 25.0 is corrected to int so mapping will still work
+    avalon_fps = api.Session.get("AVALON_FPS")
+    d, i = math.modf(float(api.Session.get("AVALON_FPS", 25)))
+    if d == 0.0:
+        avalon_fps = int(i)
+
+    fps = {'15': 'game',
+           '24': 'film',
+           '25': 'pal',
+           '30': 'ntsc',
+           '48': 'show',
+           '50': 'palf',
+           '60': 'ntscf',
+           '23.98': '23.976fps',
+           '23.976': '23.976fps',
+           '29.97': '29.97fps',
+           '47.952': '47.952fps',
+           '47.95': '47.952fps',
```

Comment:
Just wondering, should both of these values result in the same final FPS? Or is one intended to be the `df` (Drop Frame) variant? If the latter then it seems the resulting value is wrong?

Reply:
Well those are not drop frames, just way to handle rounding. I've sometime noticed that different productions specify framerates with different precision. This handle two and three decimal places as fewer or more doesn't make much sense. This could be handled in more elegant way I suppose but way one can see whats mapped to what.